test/integration/forms/action.js

```diff
@@ -728,9 +757,9 @@ context('Patient Action Form', function() {
   });
 
   specify('submitting the form', function() {
-    const formResponses = [
+    const testFormResponses = [
       getFormResponse({
-        id: '1',
+        id: uuid(),
```

Comment:
Every `getFormResponse()` returns a form response with `id: '11111'`. Which was causing issues in this test.